scripts/docker-helper/docker-status.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
Perhaps we should separate the curses work from the main PR, because curses is a bonus, but not required script.

Reply:
Really appreciate all the open questioning - thank you! I say this because I'm pushing back pretty hard on a lot of your suggestions which doesn't at all mean their bad suggestions.  They're great.  Further, asking the obvious questions are the way we improve, or point out possibly a simpler and overlooked solution. Keep 'em coming!

The goal, as I just mentioned above, of this is actually to not just wrap docker commands, or even suggest which docker commands to run, but to really help App Developer, at the highest level easily get set up.  We're overcoming docker errors that look like this (from [this ticket](https://github.com/medic/cht-core/issues/6456), one of [many](https://github.com/medic/medic-infrastructure/issues/293) like it):

```
Exception in thread Thread-7:
Traceback (most recent call last):
  File "site-packages/urllib3/connectionpool.py", line 672, in urlopen
  File "site-packages/urllib3/connectionpool.py", line 421, in _make_request
  File "<string>", line 3, in raise_from
  File "site-packages/urllib3/connectionpool.py", line 416, in _make_request
  File "http/client.py", line 1344, in getresponse
  File "http/client.py", line 306, in begin
  File "http/client.py", line 267, in _read_status
  File "socket.py", line 589, in readinto
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "site-packages/requests/adapters.py", line 449, in send
  File "site-packages/urllib3/connectionpool.py", line 720, in urlopen
  File "site-packages/urllib3/util/retry.py", line 400, in increment
  File "site-packages/urllib3/packages/six.py", line 734, in reraise
  File "site-packages/urllib3/connectionpool.py", line 672, in urlopen
  File "site-packages/urllib3/connectionpool.py", line 421, in _make_request
  File "<string>", line 3, in raise_from
  File "site-packages/urllib3/connectionpool.py", line 416, in _make_request
  File "http/client.py", line 1344, in getresponse
  File "http/client.py", line 306, in begin
  File "http/client.py", line 267, in _read_status
  File "socket.py", line 589, in readinto
urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "threading.py", line 926, in _bootstrap_inner
  File "threading.py", line 870, in run
  File "compose/cli/log_printer.py", line 217, in watch_events
  File "compose/project.py", line 510, in yield_loop
  File "compose/project.py", line 478, in build_container_event
  File "compose/container.py", line 48, in from_id
  File "site-packages/docker/utils/decorators.py", line 19, in wrapped
  File "site-packages/docker/api/container.py", line 758, in inspect_container
  File "site-packages/docker/utils/decorators.py", line 46, in inner
  File "site-packages/docker/api/client.py", line 230, in _get
  File "site-packages/requests/sessions.py", line 546, in get
  File "site-packages/requests/sessions.py", line 533, in request
  File "site-packages/requests/sessions.py", line 646, in send
  File "site-packages/requests/adapters.py", line 498, in send
requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
Exception in thread Thread-6:
Traceback (most recent call last):
  File "site-packages/urllib3/connectionpool.py", line 672, in urlopen
  File "site-packages/urllib3/connectionpool.py", line 421, in _make_request
  File "<string>", line 3, in raise_from
  File "site-packages/urllib3/connectionpool.py", line 416, in _make_request
  File "http/client.py", line 1344, in getresponse
  File "http/client.py", line 306, in begin
  File "http/client.py", line 267, in _read_status
  File "socket.py", line 589, in readinto
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "site-packages/requests/adapters.py", line 449, in send
  File "site-packages/urllib3/connectionpool.py", line 720, in urlopen
  File "site-packages/urllib3/util/retry.py", line 400, in increment
  File "site-packages/urllib3/packages/six.py", line 734, in reraise
  File "site-packages/urllib3/connectionpool.py", line 672, in urlopen
  File "site-packages/urllib3/connectionpool.py", line 421, in _make_request
  File "<string>", line 3, in raise_from
  File "site-packages/urllib3/connectionpool.py", line 416, in _make_request
  File "http/client.py", line 1344, in getresponse
  File "http/client.py", line 306, in begin
  File "http/client.py", line 267, in _read_status
  File "socket.py", line 589, in readinto
urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "threading.py", line 926, in _bootstrap_inner
  File "threading.py", line 870, in run
  File "compose/cli/log_printer.py", line 169, in tail_container_logs
  File "compose/cli/log_printer.py", line 200, in wait_on_exit
  File "compose/container.py", line 281, in wait
  File "site-packages/docker/utils/decorators.py", line 19, in wrapped
  File "site-packages/docker/api/container.py", line 1289, in wait
  File "site-packages/docker/utils/decorators.py", line 46, in inner
  File "site-packages/docker/api/client.py", line 226, in _post
  File "site-packages/requests/sessions.py", line 581, in post
  File "site-packages/requests/sessions.py", line 533, in request
  File "site-packages/requests/sessions.py", line 646, in send
  File "site-packages/requests/adapters.py", line 498, in send
requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
```

This verbosity is overwhelming and not actionable by an App Dev who's new to Medic or new to developing for the CHT (or both!!).  

Conversely, given that it may take multiple restarts which can span possibly over 10 minutes on a slow connection/slow hardware, a Bash script that uses curses to give it a dashboard like feel that intentionally obfuscates docker raw logs (`--detach` FTW!)